samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/FeatureLayers.razor

```diff
@@ -1,133 +1,54 @@
 ﻿@page "/feature-layers"
+<PageTitle>FeatureLayer</PageTitle>
```

Comment:
You have overwritten the Feature Layer's sample. Please put it back. This could be a new sample, if it makes sense to do so.